src/components/KPop/KPop.vue

```diff
@@ -661,7 +667,7 @@ $transition-speed: .3s !default;
 .fadeIn,
 .fade-leave-active,
 .fadeOut {
-  animation-duration: $transition-speed;
+  animation-duration: $tmp-animation-timing-3;
```

Comment:
question: can we just use the existing `$tmp-animation-timing-2` here and just standardize animation durations to .2s?

Reply:
Okay, fixed now